assets/pages/regular-contributions/__tests__/__snapshots__/regularContributionsReducersTest.js.snap

```diff
@@ -15,9 +15,13 @@ Object {
     "accountHolderConfirmation": false,
     "accountHolderName": "",
     "bankAccountNumber": "",
-    "bankSortCode": "",
     "formError": "",
     "isPopUpOpen": false,
+    "sortCode": Array [
```

Comment:
just out of curiosity, did you update the snapshots manually? or using our shiny yarn command `jest-update-snapshot`?

Reply:
Intellij will ask you if you want to update them when they fail

Reply:
<img width="522" alt="screen shot 2018-02-23 at 12 24 49" src="https://user-images.githubusercontent.com/181371/36594096-a7029368-1894-11e8-92fc-6c868dc7a59b.png">

Reply:
Brillant!